package.json

```diff
@@ -7,6 +7,7 @@
     },
     "description": "Converts your TSLint configuration to the closest possible ESLint equivalent.",
     "dependencies": {
+        "@typescript-eslint/eslint-plugin-tslint": "^2.3.2",
```

Comment:
Err, is this necessary?

Reply:
Adding this wasn't in my original plan.

I added it when it some of the end to end tests were failing on the [CI server](https://circleci.com/gh/typescript-eslint/tslint-to-eslint-config/1574?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link).

Reply:
Yeah, if there are separate shenanigans we can tackle them in a different PR. I don't want to block your changes here on them. :)

Reply:
I've been looking into this today and I think that this might be a symptom of an issue with the end to end tests.  I'm still getting my head around the tests, and the potential issue. 

So far it seems the `--config`` parameter is not being applied to the tests correctly. This means that the tests are validating against a committed `eslint.json` file rather than the file created by the test.
As you say probably makes sense to do under a different PR.

Reply:
If you're interested I have begun a branch in my fork to get the tests working. https://github.com/KleeUT/tslint-to-eslint-config/tree/end-to-end-tests